docs/Config.md

```diff
@@ -75,3 +75,45 @@ The secrets in this example are dummy values and won't work in an actual app.
   }
 }
 ```
+
+### Running over HTTPS
+
+You need to change change the `protocol` to `https`, and tell the server where
```

Comment:
Where you get certs for Meetup is actually in `mup-web`, since that's where the config goes.